fdbcli/MaintenanceCommand.actor.cpp

```diff
@@ -64,12 +64,19 @@ ACTOR Future<Void> printHealthyZone(Reference<IDatabase> db) {
 	}
 }
 
-// clear ongoing maintenance, let clearSSFailureZoneString = true to enable data distribution for storage
-ACTOR Future<bool> clearHealthyZone(Reference<IDatabase> db,
-                                    bool printWarning = false,
-                                    bool clearSSFailureZoneString = false) {
+} // namespace
```

Comment:
Change in this file `MaintenanceCommand.actor.cpp` is trivial.
It moves the two functions `setHealthyZone` and `clearHealthyZone` into _fdb_cli_ namespace to be used by others.